### ISSUE TITLE
Gossip strategy example not working

### ISSUE THREAD

**Issue by doughsay** [bitwalker/libcluster#22]
Just copy pasting the example gossip strategy config from the readme results in this:

```
01:02:21.787 [info] Application libcluster exited: Cluster.App.start(:normal, []) returned an error: shutdown: failed to start child: Cluster.Strategy.Gossip
    ** (EXIT) an exception was raised:
        ** (KeyError) key :topology not found in: [[topology: :gossip_example, connect: {:net_kernel, :connect, []}, disconnect: {:net_kernel, :disconnect, []}, config: [port: 45892, if_addr: {0, 0, 0, 0}, multicast_addr: {230, 1, 1, 251}, multicast_ttl: 1]]]
            (elixir) lib/keyword.ex:343: Keyword.fetch!/2
            (libcluster) lib/strategy/gossip.ex:46: Cluster.Strategy.Gossip.init/1
            (stdlib) gen_server.erl:365: :gen_server.init_it/2
            (stdlib) gen_server.erl:333: :gen_server.init_it/6
            (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
```

I can't figure out why the `opts` being passed to the gossip worker are being wrapped in an extra list like that, when this doesn't happen for the other strategies...

**Comment by bitwalker** [comment]
That's really strange...could you tell me what version of Erlang and Elixir you are running?

**Comment by doughsay** [comment]
* Erlang/OTP 20 [erts-9.0] installed via homebrew
* tried both Elixir 1.4.4 and 1.4.5

Thanks!

**Comment by doughsay** [comment]
Also tried the just released Elixir 1.5.0, still the same.

I should also mention, it works fine using the kubernetes strategy (other than it not being able to connect to the k8s api)

**Comment by bitwalker** [comment]
I checked 2.2.0 and just published 2.2.1 - both of them work on 1.4.5 and 1.5.1. Can you paste the configuration from your `config.exs`?

**Comment by doughsay** [comment]
Sorry!  I got it working.  Still not sure what was wrong, but I wasn't using 2.1, maybe that had something to do with it...
